Patches/Vanilla Armour Expanded/ThingDefs_Misc/Armor_Industrial.xml

```diff
@@ -79,36 +79,43 @@
 					<xpath>Defs/ThingDef[defName="VAE_Apparel_AdvancedVest"]/statBases/Mass</xpath>
 					<value>
 						<Bulk>5</Bulk>
-						<WornBulk>3</WornBulk>
-						<Mass>15</Mass>
+						<WornBulk>2.5</WornBulk>
+						<Mass>12</Mass>
 					</value>
 				</li>
 				<li Class="PatchOperationReplace">
 					<xpath>Defs/ThingDef[defName="VAE_Apparel_AdvancedVest"]/statBases/StuffEffectMultiplierArmor</xpath>
 					<value>
-						<StuffEffectMultiplierArmor>10</StuffEffectMultiplierArmor>
+						<StuffEffectMultiplierArmor>8</StuffEffectMultiplierArmor>
+					</value>
+				</li>
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="VAE_Apparel_AdvancedVest"]/statBases</xpath>
+					<value>
+						<ArmorRating_Sharp>4</ArmorRating_Sharp>
 					</value>
 				</li>
 				<li Class="PatchOperationReplace">
 					<xpath>Defs/ThingDef[defName="VAE_Apparel_AdvancedVest"]/statBases/MaxHitPoints</xpath>
 					<value>
-						<MaxHitPoints>175</MaxHitPoints>
+						<MaxHitPoints>150</MaxHitPoints>
 					</value>
 				</li>
 				<!-- Miscellaneous -->
 				<li Class="PatchOperationReplace">
 					<xpath>Defs/ThingDef[defName="VAE_Apparel_AdvancedVest"]/costList</xpath>
 					<value>
 						<costList>
-							<Plasteel>20</Plasteel>
+							<Plasteel>30</Plasteel>
 							<ComponentIndustrial>1</ComponentIndustrial>
+							<Cloth>30</Cloth>
```

Comment:
While making this patch, I wanted for the advanced vest to be an alternative for the composite vest, hence why I really don't want to add devilstrand to the recipe of it. It's also kind of hard to balance this apparel item because it's stuffable, which means I have to worry about the plasteel and steel versions of it. Some time soon I'll be commiting some changes to this item, but I really wish the advanced vest will never need to have devilstrand in its recipe.

Reply:
Looking over the changes between pre-update and post update, the steel advanced vest would have 12 mm RHA and 17 MPa armor, instead of 10mm RHA and 15 MPa. Steel advanced vests have better durability per material cost (steel advanced vest's 0.398 durability per material cost vs composite vest's 0.335 durability per material cost) than the composite vest's, however composite vests do have better armor values. The advanced vest's component cost was merged into its stuff cost. The other thing I tried to tackle with the latest (at the time this comment was posted) commit was plasteel armor vests vs plasteel advanced vests. Currently plasteel advanced vests will only cost 10 plasteel more for 2 mm RHA and 2 MPa extra armor and 70 extra durability, and with this should I perhaps increase the stuff or plasteel cost to account for it?

Reply:
Actually I'd say it's the opposite way around. Devilstrand is actually generally very hard to come by via traders, but every outlander settlement, exotic goods trader and orbital bulk goods trader is guarenteed to have plasteel, and given you can call in exotic goods traders manually, or travel to outlander settlements, it's much easier to get some very early on compared to devilstrand where it's entirely possible you'll never encounter any traders with it in stock since it's not a fixed option. The thing is that it takes a lot of time for the first devilstrand harvest to kick in but usually once you have a stable supply you're set.

I think an important factor is to consider if the advanced vest needs more techs than the regular armor vest does. If it does than the current cost factor is probably fine. If it doesn't then not only is a plasteel advanced vest stronger than a regular plasteel vest, but it's also slightly lighter as well since its base mass appears to be lower as well, which arguably makes the regular armor vest entirely redundant, and the composite vest close to as well.

Reply:
> I think an important factor is to consider if the advanced vest needs more techs than the regular armor vest does. If it does than the current cost factor is probably fine. If it doesn't then not only is a plasteel advanced vest stronger than a regular plasteel vest, but it's also slightly lighter as well since its base mass appears to be lower as well, which arguably makes the regular armor vest entirely redundant, and the composite vest close to as well.

The advanced vest requires the same tech as the armor vest, so I guess it's too good now. I'm guessing that the mass change should be reverted, and the plasteel cost be upped to 30?